deploy/crds/org_v1_che_crd-v1beta1.yaml

```diff
@@ -868,6 +868,38 @@ spec:
             devfileRegistryURL:
               description: Public URL to the devfile registry.
               type: string
+            devworkspaceStatus:
```

Comment:
Can you please explain to me why it's named `devworkspaceStatus` and not just `devworkspace`? I don't understand the `status` part.

Reply:
It's the status of the devworkspace. I guess `devworkspace` makes sense, too.